docs/management/index.md

```diff
@@ -423,12 +423,12 @@ The management UI shows now a username/password login form for Basic Authenticat
 
 ### Logging out of the management UI {#about-logout-workflow}
 
-RabbitMQ follows the [OpenID Connect RP-Initiated Logout 1.0](https://openid.net/specs/openid-connect-rpinitiated-1_0.html)
-specification to implement the logout workflow provided the [OpenId Connect Discovery endpoint](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfigurationRequest) returns the attribute `end_session_endpoint` on the response. It works as follows:
+RabbitMQ implements the [OpenID Connect RP-Initiated Logout 1.0](https://openid.net/specs/openid-connect-rpinitiated-1_0.html)
+specification to logout users from the management UI and from the OAuth Provider. It works as follows:
 
-  1. The user clicks on the **Logout** button,
-  2. If the management UI found an `end_session_endpoint` from the [OpenId Connect Discovery endpoint](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfigurationRequest), the management UI sends a logout request to the Idp following the [OpenID Connect RP-Initiated Logout](https://openid.net/specs/openid-connect-rpinitiated-1_0.html) specification
-  3. If the management UI did not find an `end_session_endpoint` then the user is only logged out from the management UI
+  1. The user clicks **Logout**. 
+  2. If the [OpenId Connect Discovery endpoint](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfigurationRequest) returns an `end_session_endpoint`, the management UI sends a logout request to that endpoint to close the user's session in the OAuth Provider. When the request completes, the user is also logged out from the management ui.
+  3. In the contrary, if there is no `end_session_endpoint` then the user is only logged out from the management UI.
 
```

Comment:
I don't think you need "In the contrary", I would just say "If there is no `end_session_endpoint` returned, then......